universal_basic_income.markdown

```diff
@@ -7,32 +7,46 @@
 As your technical careers progress, most of the work you will do is to automate processes that have been done manually. Our mission is to “unlock human potential.” The software we write enables it’s users to be more productive, and to direct energy toward solving increasingly difficult problems. However, some automation, especially of physical processes, can force some out of the jobs they’ve done for years. This has the potential to do more harm than good for inclusion in our society.
 
 
-## Read and Write (15 min read/10 min write)
+## Intro (5 min)
+* What is UBI? (Have propmts written on board in advance) brief intro, keep questions in mind during video
 
-As you read through theses article on universal basic income (UBI) as a response to job loss from automation, reflect on the following questions:
+## Watch (5 min)
 
+As you watch the following video on universal basic income (UBI) keep the following questions in mind.
+* https://www.youtube.com/watch?v=SQXxcjJ5ODw
+
+* Why would we need UBI?
+* What are some of the pros/cons of UBI?
 *   How would you describe the "realities" of our future economy? What constraints and opportunities will we face as a nation?
-*   What's your reaction to hearing tech leaders like Elon Musk describe UBI as a "foregone conclusion"?
-*   Write a quick elevator pitch synthesizing what you've read from these articles: what is UBI, why is it entering policy debates, what do you think people should know about automation and its affect on our workforce?
-*   Do you think it's worthwhile to be educated about automation, its affect on our workforce, and policy solutions like UBI? What does it look like to be a responsible citizen in the face of such a large, inevitable transformation? Do you think you can really have an impact -- if not, why put energy into being informed?
+*   Do you think it's worthwhile to be educated about automation, its affect on our workforce, and policy solutions like UBI? What does it look like to be a responsible citizen in the face of such a large, inevitable transformation? Do you think you 
+* If you had a UBI, would syou still work(full-time, part-time, not at all?)
```

Comment:
These are great questions.  I'd write these on the board or have slides available in order for students to keep focused on these.  Could be interesting to add a question of how UBI is going to impact your role as a developer.